### PR TITLE
issue #198: Avoid getting alerts from Code scanning

### PR DESCRIPTION
**Pull request by ivanramosnet**
Fix issue #198

### CODE REVIEW

File: __tests__/unit/git-repo.test.ts [hunk 1]
```diff
@@ -37,4 +44,33 @@ describe('GitRepo', () => {
...
+    const commitMessage = new CommitMessage(subject, body)
+
+    await expect(gitRepo.hasCommits()).resolves.toBe(false)
+
+    await gitRepo.commit(commitMessage, commitOptionsForTests())
+
+    await expect(gitRepo.hasCommits()).resolves.toBe(true)
+  })
+
+  it('should fail when the repo has not been initialized and check if it has commits', async () => {
```
Comment (josecelano) [inline __tests__/unit/git-repo.test.ts:67]: hi @ivanramosnet I did not understand the test intention well. Maybe it's only me but I would rephrase it to:

```
it should fail while checking if the repo has commits when it has not been initialized
```

**Review by josecelano — APPROVED** [review]

**Comment by josecelano** [comment]
hi @ivanramosnet can you rebase? There is a failing test because now we need the job id in the commit subject. You only need to add it to this function:

```typescript
export function dummyCommitSubjectText(): string {
  return '📝🈺: queue-name: job.id.1 job.ref.f1a69d48a01cc130a64aeac5eaf762e4ba685de7'
}
```

File: __tests__/unit/git-repo.test.ts [hunk 2]
```diff
@@ -37,4 +44,33 @@ describe('GitRepo', () => {
 
     expect(gitRepo.isInitialized()).toBe(true)
   })
+
+  it('should check if a repo has commits', async () => {
```
Comment (da2ce7) [inline __tests__/unit/git-repo.test.ts:48]: Maybe:
`initialize new git repository, check that there are no comments, add a test comment, check for the comment`

**Comment by josecelano** [comment]
Replaced by https://github.com/Nautilus-Cyberneering/git-queue/pull/235
